ibis/backends/clickhouse/tests/snapshots/test_select/test_array_join_in_subquery/out.sql

```diff
@@ -1,5 +1,5 @@
 SELECT
   t0.id IN (SELECT
     arrayJoin(t1.ids) AS ids
-  FROM way_view AS t1) AS "InColumn(id, ids)"
+  FROM way_view AS t1) AS ids
```

Comment:
Is this alias change desired?

Reply:
Good question, I'll look into what's happening.

Reply:
It looks like the operations' name changed to `ids` because of the `name` property introduced to the `Subquery` operation. I've changed that property to called `_name` since the base `name` implementation is used for naming expressions while the new `_name` is used for plucking out the single column used in an `InSubquery` operation.

```
```

Reply:
I actually just inlined the `_name` property since it's only used in `Subquery.value`.